### DISCUSSION TITLE
Loading PIL Image to QPixMap

### DISCUSSION THREAD

**Discussion by hesmith1029** [python-pillow/Pillow#8314] [Q&A] [answered]
I have a need to get PSD photos into a QLabel from PyQt5.  PIL.ImageQt would appear to be the tool. 

I keep getting errors like 'module' object is not callable.

relevant code
```python
from PIL import Image, ImageQt

im = Image.Open(filename)
imagetemp = ImageQt(im)     ----  This is the line the errors
pixmap = QPixmap.fromImage(imagetemp)
```
I have found lots of examples in StackOverflow and others on Google and this seems to be the way.

I have tried variations, such as
```python
imagetemp = ImageQt.ImageQt(im)
```
or
```python
imagetemp = PIL.ImageQt.ImageQt(im)
```
PyCharm code completion suggests all of these should work.

Can find nothing on why errors that  ImageQt is not callable.   Have tried reinstalling PyQt5 and Pillow, no joy.

Using PyCharm as my IDE.  

Any Suggestions.

**Comment by Yay295** [comment]
> `im = Image.Open(filename)`

I don't know why you're not getting an error, but `Open` should be lowercase: `im = Image.open(filename)`
Then `imagetemp = ImageQt.ImageQt(im)` should work.

**Comment by radarhere** [comment, marked as answer]
I imagine that you're using the latest Pillow version, which at the moment is 10.4.0.

PyQt5 support was removed in Pillow 10.0.0. See https://pillow.readthedocs.io/en/stable/deprecations.html#pyqt5-and-pyside2. 

I suggest you switch to PyQt6.

**Reply by hesmith1029** [reply]
Yep saw that this morning.   It is in big bold letters, but I still missed many time.   Thanks
Just for anyone else running across this.   PyQt6-tools,  where you get pyuic6, is only compatible up to Python 3.9.   So I am using PyQt6 with Python 3.9.
